examples/torch/common/utils.py

```diff
@@ -161,7 +161,7 @@ def configure_logging(sample_logger, config):
 
     nncf_log_file_handler = logging.FileHandler(osp.join(config.log_dir, NNCF_LOG_FILE_NAME))
     nncf_log_file_handler.setFormatter(logging.Formatter("%(levelname)s:%(name)s:%(message)s"))
-    from nncf.common.utils.logger import logger as nncf_logger
+    from nncf.common.logging import nncf_logger
```

Comment:
Can we have `nncf_logger` imported from `nncf` namespace directly?

Reply:
Actually, since I needed to import the logger to do the error logging in the `__init__.py`, it should already be possible to do so. However, the logger is for logging from the NNCF package code, not from the user code, so I don't think that this will bring much benefit. Let the users utilize their own logging approaches and not touch others (save for the `set_log_level` and related functions)

Reply:
Ok, I see. I make sense but I could not find `set_log_level` in the documentation. But it looks important from the debugging standpoint.